node/src/components/consensus/highway_core/highway.rs

```diff
@@ -124,6 +124,7 @@ impl<C: Context> Highway<C> {
         validators: Validators<C::ValidatorId>,
         params: Params,
     ) -> Highway<C> {
+        info!(?validators, "creating Highway instance {:?}", instance_id);
```

Comment:
```suggestion
        info!(?validators, "Creating Highway instance {:?}", instance_id);
```

Reply:
[Not according to the Coding Standards](https://casperlabs.atlassian.net/wiki/spaces/REL/pages/16842753/Coding+Standards#CodingStandards-LogmessagesandDisplayimpls)

Reply:
`InstanceId` is validators' mapping? oO

Reply:
No, but `?validators` should print the validator IDs (i.e. their public keys) in order.

Reply:
:see_no_evil: